processor/configuration/src/main/resources/schemas/sources/azure_eventhubs_source_0.1.json

```diff
@@ -0,0 +1,69 @@
+{
+  "type" : "object",
+  "additionalProperties" : false,
+  "required" : [
+    "azure_namespace_name",
+    "azure_eventhub_name",
+    "azure_shared_access_name",
+    "azure_shared_access_key",
+    "azure_blob_account_name",
+    "azure_blob_access_key",
+    "azure_blob_container_name"
+  ],
+  "properties" : {
+    "azure_namespace_name" : {
+      "title" : "Eventhubs Namespace",
+      "description" : "The Event Hubs namespace.",
+      "type" : "string"
+    },
+    "azure_eventhub_name" : {
+      "title" : "Eventhubs Name",
+      "description" : "The Event Hub name.",
```

Comment:
Reading other description fields shouldn't we use `Event Hubs` instead of `Event Hub`?

Reply:
I just copy it from Manage connector catalog.